sentinel-adapter/sentinel-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/origin/DefaultDubboOriginParser.java

```diff
@@ -0,0 +1,20 @@
+package com.alibaba.csp.sentinel.adapter.dubbo.origin;
```

Comment:
Could you please add the license header?

Reply:
Yes, I will do. ^_^